toolprovider/mise/mise.go

```diff
@@ -51,7 +59,13 @@ func (m *MiseToolProvider) Bootstrap() error {
 	fmt.Printf("Installing Mise %s...", miseVersion)
 	fmt.Println()
 
-	err := installReleaseBinary(miseVersion, m.ExecEnv.InstallDir)
+	osArch := fmt.Sprintf("%s_%s", runtime.GOOS, runtime.GOARCH)
```

Comment:
This logic already exists here: https://github.com/bitrise-io/bitrise/blob/997944818042c6954b95a30c65759a5079a671d2/toolprovider/mise/bootstrap.go#L115

Can we extract that to a shared place and just invoke it from these two places?